### ISSUE TITLE
Move to binder-examples ?

### ISSUE THREAD

**Issue by choldgraf** [minrk/binder-bokeh-server#2]
Once we get a working version that's ready for others to use, mind if I copypaste this into the binder-examples organization?

**Comment by minrk** [comment]
Yeah, probably. I was thinking I'd like to avoid having to write and install an extension here (so that doesn't need to be duplicated across repos) before doing that, but we could fix it later.
